apps/taquito-test-dapp/src/tests.ts

```diff
@@ -678,6 +702,18 @@ export const init = (
     inputType: "string",
     lastResult: { option: "none", val: false }
   },
+  {
+    id: "sign-failingNoop",
+    name: "Sign the provided payload in a failing noop",
+    description: "This test signs the payload provided by the user wrapped in a failing noop",
+    documentation: 'https://tezostaquito.io/docs/signing/#generating-a-signature-with-beacon-sdk',
```

Comment:
would be great to have a failing_noop file under doc in our website to reference here